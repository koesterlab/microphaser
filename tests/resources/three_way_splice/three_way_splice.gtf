chr19	ensembl_havana	gene	15878023	15898077	.	-	.	gene_id "ENSG00000186115"; gene_version "13"; gene_name "CYP4F2"; gene_source "ensembl_havana"; gene_biotype "protein_coding";
chr19	havana	transcript	15892373	15897729	.	-	.	gene_id "ENSG00000186115"; gene_version "13"; transcript_id "ENST00000586927"; transcript_version "2"; gene_name "CYP4F2"; gene_source "ensembl_havana"; gene_biotype "protein_coding"; transcript_name "CYP4F2-204"; transcript_source "havana"; transcript_biotype "protein_coding"; tag "cds_end_NF"; tag "mRNA_end_NF"; transcript_support_level "4";
chr19	havana	exon	15897414	15897729	.	-	.	gene_id "ENSG00000186115"; gene_version "13"; transcript_id "ENST00000586927"; transcript_version "2"; exon_number "1"; gene_name "CYP4F2"; gene_source "ensembl_havana"; gene_biotype "protein_coding"; transcript_name "CYP4F2-204"; transcript_source "havana"; transcript_biotype "protein_coding"; exon_id "ENSE00002741977"; exon_version "1"; tag "cds_end_NF"; tag "mRNA_end_NF"; transcript_support_level "4";
chr19	havana	CDS	15897414	15897611	.	-	0	gene_id "ENSG00000186115"; gene_version "13"; transcript_id "ENST00000586927"; transcript_version "2"; exon_number "1"; gene_name "CYP4F2"; gene_source "ensembl_havana"; gene_biotype "protein_coding"; transcript_name "CYP4F2-204"; transcript_source "havana"; transcript_biotype "protein_coding"; protein_id "ENSP00000465514"; protein_version "1"; tag "cds_end_NF"; tag "mRNA_end_NF"; transcript_support_level "4";
chr19	havana	start_codon	15897609	15897611	.	-	0	gene_id "ENSG00000186115"; gene_version "13"; transcript_id "ENST00000586927"; transcript_version "2"; exon_number "1"; gene_name "CYP4F2"; gene_source "ensembl_havana"; gene_biotype "protein_coding"; transcript_name "CYP4F2-204"; transcript_source "havana"; transcript_biotype "protein_coding"; tag "cds_end_NF"; tag "mRNA_end_NF"; transcript_support_level "4";
chr19	havana	exon	15895506	15895650	.	-	.	gene_id "ENSG00000186115"; gene_version "13"; transcript_id "ENST00000586927"; transcript_version "2"; exon_number "2"; gene_name "CYP4F2"; gene_source "ensembl_havana"; gene_biotype "protein_coding"; transcript_name "CYP4F2-204"; transcript_source "havana"; transcript_biotype "protein_coding"; exon_id "ENSE00003637585"; exon_version "1"; tag "cds_end_NF"; tag "mRNA_end_NF"; transcript_support_level "4";
chr19	havana	CDS	15895506	15895650	.	-	0	gene_id "ENSG00000186115"; gene_version "13"; transcript_id "ENST00000586927"; transcript_version "2"; exon_number "2"; gene_name "CYP4F2"; gene_source "ensembl_havana"; gene_biotype "protein_coding"; transcript_name "CYP4F2-204"; transcript_source "havana"; transcript_biotype "protein_coding"; protein_id "ENSP00000465514"; protein_version "1"; tag "cds_end_NF"; tag "mRNA_end_NF"; transcript_support_level "4";
chr19	havana	exon	15892529	15892582	.	-	.	gene_id "ENSG00000186115"; gene_version "13"; transcript_id "ENST00000586927"; transcript_version "2"; exon_number "3"; gene_name "CYP4F2"; gene_source "ensembl_havana"; gene_biotype "protein_coding"; transcript_name "CYP4F2-204"; transcript_source "havana"; transcript_biotype "protein_coding"; exon_id "ENSE00003725631"; exon_version "1"; tag "cds_end_NF"; tag "mRNA_end_NF"; transcript_support_level "4";
chr19	havana	CDS	15892529	15892582	.	-	2	gene_id "ENSG00000186115"; gene_version "13"; transcript_id "ENST00000586927"; transcript_version "2"; exon_number "3"; gene_name "CYP4F2"; gene_source "ensembl_havana"; gene_biotype "protein_coding"; transcript_name "CYP4F2-204"; transcript_source "havana"; transcript_biotype "protein_coding"; protein_id "ENSP00000465514"; protein_version "1"; tag "cds_end_NF"; tag "mRNA_end_NF"; transcript_support_level "4";
chr19	havana	exon	15892373	15892436	.	-	.	gene_id "ENSG00000186115"; gene_version "13"; transcript_id "ENST00000586927"; transcript_version "2"; exon_number "4"; gene_name "CYP4F2"; gene_source "ensembl_havana"; gene_biotype "protein_coding"; transcript_name "CYP4F2-204"; transcript_source "havana"; transcript_biotype "protein_coding"; exon_id "ENSE00002779498"; exon_version "1"; tag "cds_end_NF"; tag "mRNA_end_NF"; transcript_support_level "4";
chr19	havana	CDS	15892373	15892436	.	-	2	gene_id "ENSG00000186115"; gene_version "13"; transcript_id "ENST00000586927"; transcript_version "2"; exon_number "4"; gene_name "CYP4F2"; gene_source "ensembl_havana"; gene_biotype "protein_coding"; transcript_name "CYP4F2-204"; transcript_source "havana"; transcript_biotype "protein_coding"; protein_id "ENSP00000465514"; protein_version "1"; tag "cds_end_NF"; tag "mRNA_end_NF"; transcript_support_level "4";
chr19	havana	five_prime_utr	15897612	15897729	.	-	.	gene_id "ENSG00000186115"; gene_version "13"; transcript_id "ENST00000586927"; transcript_version "2"; gene_name "CYP4F2"; gene_source "ensembl_havana"; gene_biotype "protein_coding"; transcript_name "CYP4F2-204"; transcript_source "havana"; transcript_biotype "protein_coding"; tag "cds_end_NF"; tag "mRNA_end_NF"; transcript_support_level "4";
