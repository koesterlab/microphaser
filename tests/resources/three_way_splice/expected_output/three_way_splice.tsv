id	transcript	gene_id	gene_name	chrom	offset	freq	depth	nvar	nsomatic	nvariant_sites	nsomvariant_sites	strand	variant_sites	somatic_positions	somatic_aa_change	germline_positions	germline_aa_change	normal_sequence	mutant_sequence
e676ee0c506edc6R	ENST00000586927	ENSG00000186115	CYP4F2	19	15892374	0.42857142857142855	7	2	2	2	2	Reverse	15892386|15892387	15892386|15892387	|p.Arg149Gln			AGGCGTCAGCATCCGACGGTGGCGGCTCCACTTGTCACCAGCACTCAGCAGGAGCCCATC	AGGCGTCAGCATttGACGGTGGCGGCTCCACTTGTCACCAGCACTCAGCAGGAGCCCATC
f601fdd9b3518a4R	ENST00000586927	ENSG00000186115	CYP4F2	19	15892543	0.42857142857142855	17	2	2	2	2	Reverse	15892386|15892387	15892386|15892387	|p.Arg149Gln			TCCAGGAAGCTGTAGAAGAACTTGTCCTTTGGTGCAATGGCAGGCAGGCGTCAGCATCCG	TCCAGGAAGCTGTAGAAGAACTTGTCCTTTGGTGCAATGGCAGGCAGGCGTCAGCATttG
efb74d2c669e11dR	ENST00000586927	ENSG00000186115	CYP4F2	19	15892546	0.42857142857142855	17	2	2	2	2	Reverse	15892386|15892387	15892386|15892387	|p.Arg149Gln			AGGAAGCTGTAGAAGAACTTGTCCTTTGGTGCAATGGCAGGCAGGCGTCAGCATCCGACG	AGGAAGCTGTAGAAGAACTTGTCCTTTGGTGCAATGGCAGGCAGGCGTCAGCATttGACG
3e7e0b0add2f6b0R	ENST00000586927	ENSG00000186115	CYP4F2	19	15892549	0.42857142857142855	17	2	2	2	2	Reverse	15892386|15892387	15892386|15892387	|p.Arg149Gln			AAGCTGTAGAAGAACTTGTCCTTTGGTGCAATGGCAGGCAGGCGTCAGCATCCGACGGTG	AAGCTGTAGAAGAACTTGTCCTTTGGTGCAATGGCAGGCAGGCGTCAGCATttGACGGTG
8715e5f6259684cR	ENST00000586927	ENSG00000186115	CYP4F2	19	15892552	0.42857142857142855	17	2	2	2	2	Reverse	15892386|15892387	15892386|15892387	|p.Arg149Gln			CTGTAGAAGAACTTGTCCTTTGGTGCAATGGCAGGCAGGCGTCAGCATCCGACGGTGGCG	CTGTAGAAGAACTTGTCCTTTGGTGCAATGGCAGGCAGGCGTCAGCATttGACGGTGGCG
5149ad5b651f523R	ENST00000586927	ENSG00000186115	CYP4F2	19	15892555	0.42857142857142855	17	2	2	2	2	Reverse	15892386|15892387	15892386|15892387	|p.Arg149Gln			TAGAAGAACTTGTCCTTTGGTGCAATGGCAGGCAGGCGTCAGCATCCGACGGTGGCGGCT	TAGAAGAACTTGTCCTTTGGTGCAATGGCAGGCAGGCGTCAGCATttGACGGTGGCGGCT
32786878b974cd2R	ENST00000586927	ENSG00000186115	CYP4F2	19	15892558	0.42857142857142855	17	2	2	2	2	Reverse	15892386|15892387	15892386|15892387	|p.Arg149Gln			AAGAACTTGTCCTTTGGTGCAATGGCAGGCAGGCGTCAGCATCCGACGGTGGCGGCTCCA	AAGAACTTGTCCTTTGGTGCAATGGCAGGCAGGCGTCAGCATttGACGGTGGCGGCTCCA
81b0ab24931460eR	ENST00000586927	ENSG00000186115	CYP4F2	19	15892561	0.42857142857142855	17	2	2	2	2	Reverse	15892386|15892387	15892386|15892387	|p.Arg149Gln			AACTTGTCCTTTGGTGCAATGGCAGGCAGGCGTCAGCATCCGACGGTGGCGGCTCCACTT	AACTTGTCCTTTGGTGCAATGGCAGGCAGGCGTCAGCATttGACGGTGGCGGCTCCACTT
c291e40ed06065cR	ENST00000586927	ENSG00000186115	CYP4F2	19	15892564	0.42857142857142855	17	2	2	2	2	Reverse	15892386|15892387	15892386|15892387	|p.Arg149Gln			TTGTCCTTTGGTGCAATGGCAGGCAGGCGTCAGCATCCGACGGTGGCGGCTCCACTTGTC	TTGTCCTTTGGTGCAATGGCAGGCAGGCGTCAGCATttGACGGTGGCGGCTCCACTTGTC
5368888b41dee7aR	ENST00000586927	ENSG00000186115	CYP4F2	19	15892567	0.42857142857142855	17	2	2	2	2	Reverse	15892386|15892387	15892386|15892387	|p.Arg149Gln			TCCTTTGGTGCAATGGCAGGCAGGCGTCAGCATCCGACGGTGGCGGCTCCACTTGTCACC	TCCTTTGGTGCAATGGCAGGCAGGCGTCAGCATttGACGGTGGCGGCTCCACTTGTCACC
c466a13862b26a1R	ENST00000586927	ENSG00000186115	CYP4F2	19	15892570	0.42857142857142855	17	2	2	2	2	Reverse	15892386|15892387	15892386|15892387	|p.Arg149Gln			TTTGGTGCAATGGCAGGCAGGCGTCAGCATCCGACGGTGGCGGCTCCACTTGTCACCAGC	TTTGGTGCAATGGCAGGCAGGCGTCAGCATttGACGGTGGCGGCTCCACTTGTCACCAGC
85e296373d1d648R	ENST00000586927	ENSG00000186115	CYP4F2	19	15892573	0.42857142857142855	17	2	2	2	2	Reverse	15892386|15892387	15892386|15892387	|p.Arg149Gln			GGTGCAATGGCAGGCAGGCGTCAGCATCCGACGGTGGCGGCTCCACTTGTCACCAGCACT	GGTGCAATGGCAGGCAGGCGTCAGCATttGACGGTGGCGGCTCCACTTGTCACCAGCACT
0248b8235a0ce61R	ENST00000586927	ENSG00000186115	CYP4F2	19	15892576	0.42857142857142855	17	2	2	2	2	Reverse	15892386|15892387	15892386|15892387	|p.Arg149Gln			GCAATGGCAGGCAGGCGTCAGCATCCGACGGTGGCGGCTCCACTTGTCACCAGCACTCAG	GCAATGGCAGGCAGGCGTCAGCATttGACGGTGGCGGCTCCACTTGTCACCAGCACTCAG
f4b7962ba5e3229R	ENST00000586927	ENSG00000186115	CYP4F2	19	15892579	0.42857142857142855	17	2	2	2	2	Reverse	15892386|15892387	15892386|15892387	|p.Arg149Gln			ATGGCAGGCAGGCGTCAGCATCCGACGGTGGCGGCTCCACTTGTCACCAGCACTCAGCAG	ATGGCAGGCAGGCGTCAGCATttGACGGTGGCGGCTCCACTTGTCACCAGCACTCAGCAG
b59dba2f5a426bbR	ENST00000586927	ENSG00000186115	CYP4F2	19	15892582	0.42857142857142855	17	2	2	2	2	Reverse	15892386|15892387	15892386|15892387	|p.Arg149Gln			GCAGGCAGGCGTCAGCATCCGACGGTGGCGGCTCCACTTGTCACCAGCACTCAGCAGGAG	GCAGGCAGGCGTCAGCATttGACGGTGGCGGCTCCACTTGTCACCAGCACTCAGCAGGAG
38f6043cbbdad23R	ENST00000586927	ENSG00000186115	CYP4F2	19	15892585	0.42857142857142855	17	2	2	2	2	Reverse	15892386|15892387	15892386|15892387	|p.Arg149Gln			GGCAGGCGTCAGCATCCGACGGTGGCGGCTCCACTTGTCACCAGCACTCAGCAGGAGCCC	GGCAGGCGTCAGCATttGACGGTGGCGGCTCCACTTGTCACCAGCACTCAGCAGGAGCCC
6b0edb926f77acaR	ENST00000586927	ENSG00000186115	CYP4F2	19	15892588	0.42857142857142855	17	2	2	2	2	Reverse	15892386|15892387	15892386|15892387	|p.Arg149Gln			AGGCGTCAGCATCCGACGGTGGCGGCTCCACTTGTCACCAGCACTCAGCAGGAGCCCATC	AGGCGTCAGCATttGACGGTGGCGGCTCCACTTGTCACCAGCACTCAGCAGGAGCCCATC
